app/modules/zoo-api/zooapi.factory.js

```diff
@@ -4,10 +4,21 @@ require('./zooapi.module.js')
     .factory('zooAPI', zooAPI);
 
 var ApiClient = require('panoptes-client/lib/api-client');
-
+var OAuth = require('panoptes-client/lib/oauth')
```

Comment:
Out of interest, why are instances named with capital letters here, not lowercase? That naming convention usually indicates the name of a class, not an object.

Reply:
I'm not sure, when I was writing them I didn't know the convention and now I'm just sticking there for consistency, which is probably terrible 🤷‍♂️